frontend/appflowy_flutter/lib/workspace/presentation/settings/widgets/settings_language_view.dart

```diff
@@ -30,28 +30,34 @@ class SettingsLanguageView extends StatelessWidget {
   }
 }
 
-class LanguageSelector extends StatelessWidget {
+class LanguageSelector extends StatefulWidget {
```

Comment:
Why you want to switch to stateful widget? Is something broken?

Reply:
Nope, that is a refuse from the initial implementation using PopoverControllers. Restored to StatelessWidget.